charts/argo-cd/README.md

```diff
@@ -105,6 +105,10 @@ For full list of changes please check ArtifactHub [changelog].
 
 Highlighted versions provide information about additional steps that should be performed by user when upgrading to newer version.
 
+### 5.23.2
+
+This versions fixes minor typos in values.yaml.
+
```

Comment:
This can be removed. Here we list only changes, where the user needs to pay attention / change something.

List the changes in the annotation of `Chart.yaml` is enough.

Reply:
Done!